cypress.json

```diff
@@ -1,5 +1,8 @@
 {
   "baseUrl" : "http://localhost:3000",
   "defaultCommandTimeout": 10000,
-  "video" : false
+  "video" : false,
+  "env": {
+    "homeUrl": "http://localhost:3000/"
+  }
 }
```

Comment:
just observe this file and all existing keys of the object
the difference is in only `/`, so maybe you compose your assertion in a bit different way?

you can create a file kind of url composer to get required url, it might work as
```
composeUrl("/") => "http://localhost:3000/"
```
or hard-code relative path into a file
```
relative-url.js

export const home = "/"
export const checkout = "/checkout"
...
```
or just use string interpolation for your assertion
```
cy.url().should('eq', `${Cypress.config().baseUrl}/`);
```
 OR combine somehow 3 methods described above

Reply:
done